examples/platforms/utils/link_metrics.cpp

```diff
@@ -257,4 +257,11 @@ uint8_t otLinkMetricsEnhAckGetDataLen(const otMacAddress *aMacAddress)
 exit:
     return len;
 }
+
+void otLinkMetricsResetEnhAckProbing()
```

Comment:
What about calling this in `otLinkMetricsInit`? For an RCP reset, this must be called. And we can reset the date by this chance. Then we don't need to call the util API in platform code.

Reply:
otLinkMetricsInit() is only calling from examples/platforms/simulation/radio.c simulation. I don't see anywhere except this.

Reply:
All radio platforms should call this during initialization to initialize the noisefloor. The value depends on the platform.